app/assets/javascripts/admin/views/editions/_form.js

```diff
@@ -121,6 +91,147 @@
       })
     },
 
+    toggleWorldNewsStoryVisibility: function toggleWorldNewsStoryVisibility() {
+      // only toggle fields when news article type is editable
+      if ( $( "select#edition_news_article_type_id" ).length == 0 ) return;
+
+      var nonWorldNewsStoryFieldsets = this.findFieldsets(this.nonWorldNewsStoryFieldSelectors())
+      var worldNewsStoryFieldsets = this.findFieldsets(this.worldNewsStoryFieldSelectors())
+
+      var _this = this
```

Comment:
As discussed, I'll leave it as `_this` for now, as there are more references to it through the rest of the code base